shared/cfg/guest-os/Linux/RHEL/6.9/ppc64.cfg

```diff
@@ -0,0 +1,16 @@
+- ppc64:
+    del grub_file
+    vm_arch_name = ppc64
+    image_name += -ppc64
```

Comment:
@luckyh it makes sense to keep the ppc64 consistent with the x86_64 configuration, but it doesn't seem consistent with *other* guest OS definitions.  The only reference that I see to the `guest_name` parameter is here:

https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_test/libvirt.py#L1806

And I don't even see that function being used on tp-qemu.  I'll actually remove this on v2.